docker-bake.hcl

```diff
@@ -0,0 +1,128 @@
+variable "GO_VERSION" {
+  default = "1.17.0"
+}
+variable "BIN_OUT" {
+  default = "./bin"
+}
+variable "RELEASE_OUT" {
+  default = "./release-out"
+}
+
+// Special target: https://github.com/docker/metadata-action#bake-definition
+target "docker-metadata-action" {
+  tags = ["docker/buildx-bin:local"]
+}
+
+target "_common" {
+  args = {
+    GO_VERSION = GO_VERSION
+    BUILDKIT_CONTEXT_KEEP_GIT_DIR = 1
+  }
+}
+
+group "default" {
+  targets = ["binaries"]
+}
+
+group "validate" {
+  targets = ["lint", "vendor-validate", "docs-validate"]
+}
+
+target "lint" {
+  inherits = ["_common"]
+  dockerfile = "./hack/dockerfiles/lint.Dockerfile"
+  output = ["type=cacheonly"]
+}
+
+target "vendor-validate" {
```

Comment:
I know you are trying to group things together but this order "vendor-validate" looks so unnatural to me compared to "validate-vendor". I feel like I would need to check this file every time for the target name if we leave it like this.

Reply:
Yes, I was wondering about that too. Sgtm to use `validate-xxx`.